nautobot/dcim/views.py

```diff
@@ -4255,6 +4256,105 @@ class SoftwareImageFileUIViewSet(NautobotUIViewSet):
     serializer_class = serializers.SoftwareImageFileSerializer
     table_class = tables.SoftwareImageFileTable
 
+    object_detail_content = object_detail.ObjectDetailContent(
+        panels=(
+            object_detail.ObjectFieldsPanel(
+                section=SectionChoices.LEFT_HALF,
+                weight=100,
+                fields="__all__",
+            ),
+        ),
+        extra_tabs=(
+            object_detail.DistinctViewTab(
+                weight=700,
+                tab_id="device_types",
+                url_name="dcim:softwareimagefile_device_types",
+                label="Device Type",
```

Comment:
```suggestion
                label="Device Types",
```

Similar suggestion on the other tabs below.

Reply:
Updated as requested, should be resolved, let me know if I missed anything